src/components/Layout/MainNavbar/MainNavbar.tsx

```diff
@@ -29,8 +29,8 @@ type Props = {
   toggleSidebar?: () => void;
   reconnect?: () => void;
```

Comment:
reconnect is not getting used anymore, so it should also be removed from props, no?

Reply:
Agree. I removed it from the child and not from the parent component.